### ISSUE TITLE
Dendron Paste Image: Not working in .devcontainer

### ISSUE THREAD

**Issue by 8brandon** [dendronhq/dendron#749]
## Please select the area your Bug applys to. (Multiple selections are Possible)

- [ ] Onboard. Initial vault setup and import/export pods 
- [x ] Create. Note creation, lookup, snippets and templates
- [ ] Retrieve. Backlinks, references, graph view
- [ ] Structure. Refactoring, multi-vault and schemas
- [ ] Publish. Sharing your repo with the world
- [ ] Misc (Choose this if your not sure)

## To Reproduce

1. Open Your Dendron workspace in a .devcontainer in VSCode. I chose the Open In Volume option but I think this happens with any option. I also chose the VSCode Image Node.js & Typescript.
2. Copy image to clipboard on MacOs
3. Run the Dendron Paste Image Command
4. Error Message pops up saying `You need to install Xclip`
5. exec ` sudo apt install xclip`
6. Run the Dendron Paste Image Command
7. Error Message pops up `There is not an image in the clipboard.`

## Expected behavior

Paste Markdown Image/ Add image to assets

## Screenshots

<img width="810" alt="Screenshot 2021-05-22 at 21 13 49" src="https://user-images.githubusercontent.com/34548660/119238442-9bbca800-bb42-11eb-9fbb-61266bd38082.png">

## Desktop (please complete the following information)

- OS: MacOs
- Version 11.2

**Comment by 8brandon** [comment]
This seems potentially relevant.
https://code.visualstudio.com/docs/remote/troubleshooting#_clipboard-does-not-work

**Comment by kpathakota** [comment]
Yeah, pretty sure we *do* use clipboardy in which case, hopefully this is a quick fix to support remote installs of Dendron.

**Comment by 8brandon** [comment]
> Yeah, pretty sure we _do_ use clipboardy in which case, hopefully this is a quick fix to support remote installs of Dendron.

Just looked and it seems the paste image functionality is using this library?
https://github.com/mushanshitiancai/vscode-paste-image

**Comment by kevins8** [comment]
Yes
